tinywallet/tinywallet/screens.py

```diff
@@ -848,12 +831,13 @@ def withCallback(self, callback, prompt="Password"):
         on form submission.
 
         Args:
-            callback (func(str, ...)): A function to receive the users password.
+            callback (func(str, ...)): A function to receive the user's
```

Comment:
You can remove the `...`. Used to allow passing arbitrary additional arguments, but no longer does.